### PR TITLE
Remove need for deploy-proxy.sh on each node

### PR DESCRIPTION
**Pull request by paololim**

### CODE REVIEW

File: deploy.rb [hunk 1]
```diff
@@ -67,7 +67,14 @@ def latest_tag(owner,repo)
 
 # Installs and starts software
 def deploy(node, version)
-  cmd = "ssh #{node} ./deploy-proxy.sh #{version}"
+  docker_commands = [
+    "docker stop `docker ps -q`",
```
Comment (benwaffle) [inline deploy.rb]: let's be consistent wrt subshells - let's use only `$(...)` instead of backticks

**Comment by benwaffle** [comment]
And same for line 72 please :)

**Comment by benwaffle** [comment]
I should've used the multiline comment

File: deploy.rb [hunk 2]
```diff
@@ -67,7 +67,14 @@ def latest_tag(owner,repo)
 
 # Installs and starts software
 def deploy(node, version)
-  cmd = "ssh #{node} ./deploy-proxy.sh #{version}"
+  docker_commands = [
+    "docker stop $(docker ps -q)",
+    "docker rmi -f $(docker images -q)",
+    "docker rm $(docker ps -qa --no-trunc --filter 'status=exited')",
```
Comment (benwaffle) [inline deploy.rb]: I think these single quotes will conflict with the single quotes in the `ssh` command on line 77

Comment (paololim) [reply deploy.rb]: I just copy/pasted this from the current deploy-proxy.sh

**Pull request closed without merging**